src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
Since you're using the `Lottery` instance multiple times, it would be more efficient to declare it once outside of the loop or the repetitive method calls.